client/src/components/ElementEditor/Element.js

```diff
@@ -20,7 +20,7 @@ class Element extends Component {
   /**
    * Take the user to the GridFieldDetailForm to edit the record
    */
-  handleClick() {
```

Comment:
We don't need this any more =)

Reply:
Do you say that because this functionality is moving into the caret button in the header? And we don't want to replicate this if the user clicks anywhere on the element?!

Reply:
I think it's mentioning that you've added `event` but you're not using it anymore 🙂

Reply:
ah, i removed that since.